### ISSUE TITLE
Optative in τίθημι: Present System Middle-Passive

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios-core#77]
Some endings in the Optative for verbpdgm29 τίθημι: Present System Middle-Passive table are missing in Morpheus, except for the first person singular and the first person plural, 3rd person dual.
τιθεῖτο, the 3rd person singular optative goes to an other table as its stemtype is ew_pr and not emi_pr.
I did also check the Goodell grammar and there are the same endings as in Mastronarde tables. Morpheus or Mastronarde?

**Comment by balmas** [comment]
@monzug I'm a little lost with this item.  Are there any inflection table fixes here or are these all problems with Morpheus not recognizing forms?

**Comment by monzug** [comment]
Morpheus issues mainly.
except for τίθημι forms, is there anything we can do to match ἔθηκας
ἔθηκα
ἔθηκε with verbpdgm30?  and τιθεῖτο

**Comment by balmas** [comment]
added rules for tables 28, 30 and 31 to match on lemma τίθημι regardless of stemtype

**Comment by monzug** [comment]
beautiful. tested in Alpheios Reading Tools 3.1.0 build qa.20200330265
Alpheios Components 3.3.0-qa.20200327479

whatever form is not available in Morpheus is still missing.

**Comment by balmas** [comment]
>@balmas, whatever form is not available in Morpheus is still missing. Do we want to keep track of these missing forms in a general Morpheus GitHub?

Yes @monzug, please add issues for them to https://github.com/alpheios-project/morpheus/issues
